users/views.py

```
@@ -79,4 +83,61 @@ def post(self, request, *args, **kwargs):
             return Response(
                 {"message": "Invalid token or already logged out."}, 
                 status=status.HTTP_400_BAD_REQUEST
-            )
\ No newline at end of file
+            )
+
+    # --즐겨찾기--
+class FavoriteViewSet(viewsets.ModelViewSet):
+    serializer_class = FavoriteSerializer
+    permission_classes = [IsAuthenticated] # 로그인한 사용자만 접근 가능
```

Ruff (0.14.11)

Redefinition of unused `permission_classes` from line 4: `permission_classes` redefined here

(F811)